app/org/sagebionetworks/bridge/services/StudyServiceImpl.java

```diff
@@ -190,6 +190,7 @@ public Study updateStudy(Study study, boolean isAdminUpdate) {
         if (!isAdminUpdate) {
             study.setMaxNumOfParticipants(originalStudy.getMaxNumOfParticipants());
             study.setHealthCodeExportEnabled(originalStudy.isHealthCodeExportEnabled());
+            // TODO: syoung 11/16/2015 Should the minSupportedVersion require admin privileges? I'm thinking No.
```

Comment:
No, any study developer can change this. It's not a policy on our side, it's on their developers.